### PR TITLE
Update mouse natives

### PR DESCRIPTION
**Pull request by freedy69**
Added examples and changed descriptions

### CODE REVIEW

File: HUD/IsMouseRolledOverInstructionalButtons.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+```c
+// 0x3D9ACB1EB139E702
+cs_type(Any) BOOL IS_MOUSE_ROLLED_OVER_INSTRUCTIONAL_BUTTONS();
+```
+
+Note: The buttons need to support mouse (with the TOGGLE_MOUSE_SUPPORT scaleform movie method) for it to return true.
+
+## Return value
+True if the mouse cursor is hovering above instructional buttons.
```
Comment (PsychoShock) [inline HUD/IsMouseRolledOverInstructionalButtons.md:15]: When the native name is obvious, no return value description is needed.

File: HUD/SetMouseCursorThisFrame.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+aliases: ["_SHOW_CURSOR_THIS_FRAME", "_SET_MOUSE_CURSOR_ACTIVE_THIS_FRAME"]
+---
+## SET_MOUSE_CURSOR_THIS_FRAME
+
+```c
+// 0xAAE7CE1D63167423
+void SET_MOUSE_CURSOR_THIS_FRAME();
+```
+
+Shows the GTA cursor for one frame.
```
Comment (PsychoShock) [inline HUD/SetMouseCursorThisFrame.md:12]: > Sets the mouse pointer to be active on current frame

Would be better.

**Review by PsychoShock — COMMENTED** [review]
Hello,

I have left two comments. Nice work.

File: HUD/IsMouseRolledOverInstructionalButtons.md (hunk 1 above)
Comment (freedy69) [reply HUD/IsMouseRolledOverInstructionalButtons.md:15]: Alright

**Comment by PsychoShock** [comment]
Can you squash everything btw please? :)

**Comment by freedy69** [comment]
> Can you squash everything btw please? :)

what do u mean?

**Comment by PsychoShock** [comment]
Please follow this [doc](https://docs.fivem.net/docs/contributing/git/squash-guide/).

**Comment by freedy69** [comment]
> Please follow this [doc](https://docs.fivem.net/docs/contributing/git/squash-guide/).

Okay thanks! I'll do it tomorrow :)

File: HUD/SetMouseCursorStyle.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+```c
+// 0x8DB8CFFD58B62552
+void SET_MOUSE_CURSOR_STYLE(int style);
+```
+
+Changes the mouse style.
+
+Available styles are:
+| ID  | Description         |
+| :-: | :------------------:|
```
Comment (PsychoShock) [inline HUD/SetMouseCursorStyle.md:16]: Not sure if this will parse correctly. Pretty sure tables in MD need a space on `---:|` -> `---: |`

Comment (freedy69) [reply HUD/SetMouseCursorStyle.md]: Done!

**Review by 4mmonium — APPROVED** [review]
LGTM, approved, thank you! 😊

**Pull request merged**